send_rx_every_page_before_render_hook.php

```diff
@@ -0,0 +1,52 @@
+<?php
+	return function($project_id){
+        parse_str($_SERVER['QUERY_STRING'], $qs_params);
+     
+	    if($qs_params['page'] == 'patient_screening_info'){
```

Comment:
Before checking the instrument, we should check for the page we are (e.g. `PAGE == 'DataEntry/index.php'`).